app/src/androidTest/java/com/duckduckgo/app/autocomplete/api/AutoCompleteApiTest.kt

```diff
@@ -48,31 +45,94 @@ class AutoCompleteApiTest {
     }
 
```

Comment:
nit: Since we are changing this file, would you mind changing `initMocks` which is now deprecated to `openMocks' ?